### PR TITLE
#21: Added an example for connecting from AWS Sagemaker to an Exasol database

### LINKED ISSUE #21: Add SageMaker connect to exasol example
Add an example for connecting from AWS Sagemaker to Exasol.

### PR DESCRIPTION
**Pull request by jakobbraun**
Closes #21

### CODE REVIEW

File: tutorials/machine-learning/python/sagemaker/ConnectSagemakerToExasol.ipynb [hunk 1]
```diff
@@ -0,0 +1,150 @@
+{
```
Comment (tkilias) [inline tutorials/machine-learning/python/sagemaker/ConnectSagemakerToExasol.ipynb:1]: <p>might be better to define it his way</p><p> </p><p><code>EXASOL_HOST = "&lt;database_host&gt;"</code></p><p><code>EXASOL_PORT = "8563" # change if needed</code></p><p><code>EXASOL_CONNECTION = "{host}:{port}".format(host=EXASOL_HOST, port=EXASOL_HOST) # change</code></p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/exasol/data-science-examples/pull/22/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

File: tutorials/machine-learning/python/README.md [hunk 2]
```diff
@@ -9,6 +9,10 @@ This section contains tutorials with the Python Programming Language. We are goi
   
   [Scikit-learn](https://scikit-learn.org/stable/) is a free software machine learning library for the Python programming language. It features various classification, regression and clustering algorithms including support vector machines, random forests, gradient boosting, k-means and DBSCAN, and is designed to interoperate with the Python numerical and scientific libraries NumPy and SciPy. Its scalability of the training is typically limited. Out-of-core learning is not for all algorithms available, such that the usage of these algorithms is limited by the available main memory. Scikit-learn supports parallel execution through python multi-processing and linear algebra libraries. Distributed training and GPU acceleration is not out of the box available. You can find more details about scalability [here](https://scikit-learn.org/stable/modules/computing.html).
 
-### Prerequisites
+* [AWS Sagemaker](sagemaker)
```
Comment (tkilias) [inline tutorials/machine-learning/python/README.md:12]: will the tutorial about Sagemaker in general or for the time being only for Sagemaker notebooks

**Review by tkilias — COMMENTED** [review]
I think, we need also an explanation how you enable the Notebook instance to access the Exasol Cluster.

File: tutorials/machine-learning/python/README.md (hunk 2 above)
Comment (jakobbraun) [reply tutorials/machine-learning/python/README.md:19]: In the future more - in this PR only how to connect. I added a list and will extend it with the next PRs.

**Review by tkilias — APPROVED** [review]
Looks good

**Pull request merged**